src/R4MvcHostApp/Areas/Admin/Controllers/UsersController.cs

```diff
@@ -3,15 +3,37 @@
 using System.Linq;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Rendering;
 
 namespace R4MvcHostApp.Areas.Admin.Controllers
 {
     [AdminArea]
     public partial class UsersController : Controller
     {
+
+        static UsersController()
+        {
+            ModelUnbinderHelpers.ModelUnbinders.Add(typeof(Index2ViewModel), new R4Mvc.ModelUnbinders.SimplePropertyModelUnbinder());
+        }
+
         public virtual IActionResult Index()
         {
             return View();
         }
+
+        public virtual IActionResult Index2(Index2ViewModel model)
+        {
+            model = new Index2ViewModel { Id = "hello", Value = 10 };
+
+            return View(model);
+        }
+    }
+
+    public class Index2ViewModel
```

Comment:
Same with the model. It should be in the Models folder, alongside the other models :)

Reply:
Ok, sorry i never use MVC folder structure, i've always used feature folders 😄